package.json

```diff
@@ -14,7 +14,7 @@
     "clean:build": "rm -rf packages/*/lib",
     "clean:coverage": "rm -rf coverage",
     "clean:modules": "lerna clean --yes",
-    "lint": "eslint --cache packages/**/src",
+    "lint": "eslint --cache packages/**/src/*.js",
```

Comment:
Shouldn't this be:

```suggestion
    "lint": "eslint --cache packages/**/src/**/*.js",
```

Reply:
@olistic  packages/**/src/**/*.js works.

From what I understand , if in future there need to be more directories inside src folder this is way to go